tests/test_flows.py

```diff
@@ -978,6 +980,23 @@ def test_topology(self):
         self.check_log(self.repeater1, "agent", r"TOPOLOGY_QUERY_MESSAGE")
 
 
+    def test_beacon_report_query(self):
+
+        sta_mac = '00:01:02:aa:bb:ee'
+        repeater = self.repeater1
+        mac_repeater = self.mac_repeater1
+
+        debug("Connect dummy STA to wlan0")
+        self.send_bwl_event(repeater, "wlan0", "EVENT AP-STA-CONNECTED {}".format(sta_mac))
+
+        # send beacon query request
+        debug("Sending beacon report query to repeater:")
+        request='{mac} 0x73 0xFFFFFFFFFFFF 0x02 0x00 0x01 0x03 0x73 0x24 0x30'.format(mac=sta_mac)
+        debug(request)
```

Comment:
Maybe it would be better to extend dev_send_1905 to always print the command if debug is enabled.

Reply:
I vote yes!